integrations/mcp/examples/mcp_http_client.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
```

Comment:
Since there are several useful examples, would it make sense to create a README.md file in integrations/mcp/examples, which briefly introduces and links the different examples?